APOLLO-22344 - Fix Security Issues

athana opened the pull request.

janezomgzomg committed:
* remove eval fn in timezone.js

janezomgzomg committed:
* remove console.log

athana committed:
Fixed the security issue by removing a function to load scripts from Banana.
NOTES: This function seems to be unused anywhere.

athana committed:
Fix security issue in angular-dragdrop.js
by removing the use of eval().

athana committed:
Fix the security issue in require.js
by removing the use of eval().

athana committed:
Changed version to 1.6.23a in dasheditor.html
to indicate that it includes the security fixes.

athana committed:
Add files in dist/*

athana committed:
Update README.md

janezomgzomg reviewed (approved):
This looks good, I don't see any red flags in sonarqube either! :shipit: 🚢

athana committed:
Changed version from 1.6.23a to 1.6.24
Because we need to publish to npm with the proper version number.
Update README.md
